profile_library/eq-3/HmIP-FROLL/model.json

```diff
@@ -17,7 +20,20 @@
       "closing": 0
     }
   },
-  "created_at": "2024-09-06T18:37:40",
+  "sub_profile_select": {
+    "matchers": [
+      {
+        "type": "entity_state",
+        "entity_id": "cover.{{source_object_id}}",
```

Comment:
I'll make some improvement for this in the near future. When omitting it should just take the current source entity.
Let's do it like this for now.

Reply:
Sorry, I do you mean that you will change the sub profile selection in such a case to automatically choose the profile depending on the state?